Entities/AtomGroup.cpp

```diff
@@ -1451,10 +1385,10 @@ float AtomGroup::Travel(Vector &position,
 
                 // Step back all atoms that hit MO:s during this step iteration.
                 // This is so we aren't intersecting the hit MO anymore.
-                for (mapMOItr = hitMOAtoms.begin(); mapMOItr != hitMOAtoms.end(); ++mapMOItr)
+                for (map<MOID,std::list<Atom*>>::value_type mapMOItr : hitMOAtoms)
```

Comment:
Same variable name issue here and the for loop inside. I'm not sure of names for these though, would need to better understand the context.
Also, you can probably delete the predefined iterator `mapMOItr` if it's not used, wherever it's defined, if you haven't already. Same applies to other unused predefined iterators, if there are any.

Reply:
Yes, predefined iterators have already been deleted.